# Add hashbang for python to allow the main script to be executed directly
# Author: Fokke Dijkstra
# University of Groningen, NL
diff -ru pcangsd-0.97/pcangsd.py pcangsd-0.97.patched/pcangsd.py
--- pcangsd-0.97/pcangsd.py	2018-12-09 10:37:34.000000000 +0100
+++ pcangsd-0.97.patched/pcangsd.py	2019-02-25 12:55:47.000000000 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 """
 PCAngsd Framework: Population genetic analyses for NGS data using PCA. Main caller.
 """
